.pre-commit-config.yaml

```diff
@@ -40,3 +40,4 @@ repos:
     rev: v6.14.4
     hooks:
       - id: ansible-lint
+        entry: env ANSIBLE_ROLES_PATH="$ANSIBLE_ROLES_PATH:test/roles" ansible-lint
```

Comment:
Why do we need to define ANSIBLE_ROLES_PATH here?

Reply:
If you we don't define roles path ansbile lint with fail with plugin not found error. couldn't figure out how to solve it 

Example:
`syntax-check[specific]: the role 'ec2plugin' was not found in /Users/wadhahmahroug/Projects/opensource/molecule-plugins/test/roles/ec2plugin/tests/roles:/Users/wadhahmahroug/.cache/ansible-compat/5f579b/roles:/Users/wadhahmahroug/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/Users/wadhahmahroug/Projects/opensource/molecule-plugins/test/roles/ec2plugin/tests
test/roles/ec2plugin/tests/test.yml:5:7`

Reply:
did you try changing to the FQDN and add the collection providing it in the requirements.yml file ?

Reply:
Please ignore my comment, I misread the error message.